### PR TITLE
{vis}[foss/2018a] OpenCV 3.4.1 w/ Python {2.7.14, 3.6.4} (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
Needs PR ~~#6038~~ -> ~~ #5827~~
Based on PR ~~#5931~~

### CODE REVIEW

**Comment by RvDijk** [comment]
@boegel @hajgato For me it is fine to go with only one version of OpenCV, shall I close #5931 (older version) then?

**Comment by boegel** [comment]
@RvDijk Make sense to me, let's go forward with this one...

File: easybuild/easyconfigs/o/OpenCV/OpenCV-3.4.1-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [
+    'https://github.com/Itseez/opencv/archive/',
+    # The Hash is version dependent! see 3rdparty/ippicv/downloader.cmake
+    'https://raw.githubusercontent.com/Itseez/opencv_3rdparty/81a676001ca8075ada498583e4166079e5744668/ippicv',
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-3.4.1-foss-2018a-Python-3.6.4.eb]: @hajgato This commit should be updated too:

```
$ grep COMMIT opencv-3.4.1/3rdparty/ippicv/ippicv.cmake
  set(IPPICV_COMMIT "dfe3162c237af211e98b8960018b564bc209261d")
                 "https://raw.githubusercontent.com/opencv/opencv_3rdparty/${IPPICV_COMMIT}/ippicv/"
```

Comment (hajgato) [reply easybuild/easyconfigs/o/OpenCV/OpenCV-3.4.1-foss-2018a-Python-3.6.4.eb]: @boegel oops, I overlooked this.

**Comment by hajgato** [comment]
@boegel @RvDijk ippicv updated and Python 2 version added.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2645.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6ebb32dc0dbd983f5945cdc88c8e6ab9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2061.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4f9b34bf3447372a3d2ba91213382249 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**